### DISCUSSION TITLE
Choosing a theme for HNN-GUI

### DISCUSSION THREAD

**Discussion by ntolley** [jonescompneurolab/hnn-core#767] [Polls]
As the title states, I feel like the current banner color for HNN-GUI is a bit unappealing. The original HNN logo is a mixture of blue/purple of different shades if we want to stay with the current branding:

![image](https://github.com/jonescompneurolab/hnn-core/assets/55253912/47be3c83-868b-45f3-8ced-6c45e43dacfa)

I figure the best way to resolve this is with a vote! Feel free to pick your favorite or propose any colors you'd like to see added. 

# Original (no change) :1st_place_medal: 
![image](https://github.com/jonescompneurolab/hnn-core/assets/55253912/52df43ed-46ed-4ef1-a646-cd55926f5fd2)

# Different shade of purple :smiling_imp: 
![image](https://github.com/jonescompneurolab/hnn-core/assets/55253912/e4a9a2fe-400a-4b88-b6e8-ec38180038b4)

# Grey :moyai: 
![image](https://github.com/jonescompneurolab/hnn-core/assets/55253912/a1b66dbb-9314-407b-a2ce-847b490d336e)

# Dark Blue :ocean: 
![image](https://github.com/jonescompneurolab/hnn-core/assets/55253912/74c76dae-99e3-49fa-a46b-3c04b70c9c96)

# Light Blue :droplet: 
![image](https://github.com/jonescompneurolab/hnn-core/assets/55253912/d1f8b8c2-2b2d-4450-9c44-bac661355aa2)

# Green :evergreen_tree: 
![image](https://github.com/jonescompneurolab/hnn-core/assets/55253912/4ba91811-9d55-47a3-9cf8-c8f62c887a2b)

# Red :vampire: :drop_of_blood: 
![image](https://github.com/jonescompneurolab/hnn-core/assets/55253912/b7ea3ee9-1501-41e0-9065-cc9e1b9ef19e)

**Comment by dylansdaniels** [comment]
The current, neon-ish purple in HNN GUI isn't one of the colors in the HNN logo.

I can look up the Hex codes for the logo if we want to add those into consideration?

**Reply by jasmainak** [reply]
No strong opinions :) all colors are good!

**Reply by ntolley** [reply]
Seems the darker purple is becoming the crowd favorite. In any case here are shots for the other letters that are close to purple:
![image](https://github.com/jonescompneurolab/hnn-core/assets/55253912/669d594e-7677-4b0b-90c9-2c6f946315a1)

![image](https://github.com/jonescompneurolab/hnn-core/assets/55253912/0a5b43f1-c403-4e6d-bd68-75488809c0fb)

Perhaps a final vote between these two and the new purple contender is warranted

**Reply by jasmainak** [reply]
I like the second one better :) Perhaps the banner/font size needs to be slightly bigger so the letters are clearer? Or abbreviate to HNN? I'm sure Dylan can work his magic!

**Reply by dylansdaniels** [reply]
I like the second one as well! And yes, I'd be happy to play around with font, size, etc. :) It may have to be once the progress report is in next Friday
